### ISSUE TITLE
Rakefile needs update with for rdoc

### ISSUE THREAD

**Issue by sachinmaharjan** [twitter/scala-bootstrapper#34]
Need to change the following line  for "rake build' to work
require 'rake/rdoctask'
with 
require 'rdoc/task'

**Comment by geekman-rohit** [comment]
getting same error? @sachinmaharjan did you find a solution?

**Comment by sachinmaharjan** [comment]
@geekman-rohit 

The above post that I made is the solution: 

Need to change the following line for "rake build' to work
require 'rake/rdoctask'
with 
require 'rdoc/task'

**Comment by geekman-rohit** [comment]
Yes I figured that, but I am getting another error.. but now when I do gem install gem install pkg/scala-bootstrapper-*.gem  and then try to run scala-bootstrapper I get a command not found error. no executable scala-bootstrapper is being created? I thought it might be because I made the change in Rakefile??
